interview-questions.md

```diff
@@ -8,36 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: This is a problem that can be easily fixed. The foreign key is simply called belongs_to so for this example in the Student model it would be belongs_to :cohort
+
+Researched answer: To fix this you would run a rails g statement for the model. then add-in your foreign key into rails g and run rails db:migrate. Then you can add has_many :students in cohort.rb and belongs_to :cohort in students.rb.
```

Comment:
The rails command that will allow you to modify the schema is `$ rails db:migrate`.